config/crd/bases/cluster.ytsaurus.tech_ytsaurus.yaml

```diff
@@ -14893,6 +14893,10 @@ spec:
                 type: object
               oauthService:
                 properties:
+                  disableUserCreation:
+                    description: If DisableUserCreation is set, proxies will NOT create
+                      non-existing users with O
```

Comment:
```suggestion
                      non-existing users with OAuth authentication
```

Reply:
weird, will check, though it is auto-generated from comment string, I can't fix it here, it will be diff next time

Reply:
https://github.com/ytsaurus/ytsaurus-k8s-operator/pull/418